docs/topics/proc_specify-a-custom-logging-framework-vertx.adoc

```diff
@@ -0,0 +1,24 @@
+
+[#_specify_custom_logging_framework_vertx]
+= Specify a custom logging framework
+
+If you don’t want {VertX} to use `java.util.logging` for logging, you can configure it to use a different logging framework, for example, `Log4J` or `SLF4J`:
+
+.  Set the `vertx.logger-delegate-factory-class-name` system property, where `class-name` corresponds to the name of the class that implements the `LogDelegateFactory` interface.
```

Comment:
That's wrong. You set _the value_ of the `vertx.logger-delegate-factory-class-name` system property _to_ a name of the class that implements `LogDelegateFactory` interface.

Reply:
fixed:)

Reply:
fixed:)